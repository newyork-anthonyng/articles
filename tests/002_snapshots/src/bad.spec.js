it('should have useless snapshot', () => {
  const a = () => 0;

  expect(a).toMatchSnapshot();
});
