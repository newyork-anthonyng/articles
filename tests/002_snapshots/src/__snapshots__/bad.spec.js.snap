// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have useless snapshot 1`] = `[Function]`;
